src/Types/Parameter.cs

```diff
@@ -33,14 +33,41 @@ public sealed record Parameter(byte[] Param) : IEquatable<Parameter>
     /// <summary>
     /// Copies the parameters to a byte array which has the length preprended.
     /// </summary>
-	public byte[] ToBytes()
+    public byte[] ToBytes()
     {
         using var memoryStream = new MemoryStream((int)this.SerializedLength());
         memoryStream.Write(Serialization.ToBytes((ushort)this.Param.Length));
         memoryStream.Write(this.Param);
         return memoryStream.ToArray();
     }
 
+    /// <summary>
+    /// Create a parameter from JSON representation using the smart contract module schema for a smart contract update transaction.
+    /// </summary>
+    /// <param name="moduleSchema">The smart contract module schema.</param>
+    /// <param name="contractName">The name of the contract.</param>
+    /// <param name="functionName">The name of entrypoint of the smart contract.</param>
+    /// <param name="jsonParameter">The UTF8 encoding of the JSON representation of the smart contract parameter.</param>
+    public static Parameter UpdateJson(
+        VersionedModuleSchema moduleSchema,
```

Comment:
Is there a reason why parameter schemas are not supported?

We advertise those since they allow you to interact with a "generic CIS2 contract".

Reply:
I have added a `FromJson` as well, which takes a schema type and not the whole module schema.